commands/util.go

```diff
@@ -203,9 +203,23 @@ func driversForNodeGroup(ctx context.Context, dockerCli command.Cli, ng *store.N
 						logrus.Error(err)
 					}
 				}
+
+				tryToUseKubeConfigInCluster := false
 				if kcc == nil {
-					kcc = driver.KubeClientConfigInCluster{}
+					tryToUseKubeConfigInCluster = true
+				} else {
+					if _, err := kcc.ClientConfig(); err != nil {
```

Comment:
Wouldn't hurt to logrus.Debug(err) here?

Reply:
i found kcc not be nil in cluster pod.

but not found other way to check local kube config exists.

not sure which logrus.Debug(err) you means? I haven't notice logrus.Debug in Client Config

Reply:
I thought printing the err with `logrus.Debug(err)` might be useful for debugging potential failures, but either is fine to me